### ISSUE TITLE
Mute cheat

### ISSUE THREAD

**Issue by ShRules** [TotalFreedom/TotalFreedomMod#1874]
U can bypass /mute by relogging...
It'll be nice if someone fix this 

~ShRules

**Comment by Elephants111** [comment]
@JeromSar

**Comment by DUFCLiam** [comment]
@JeromSar This would be a bug.

**Comment by JeromSar** [comment]
@DUFCLiam No, the behaviour is intentional. It could however, be improved.

**Comment by ghost** [comment]
@JeromSar they can bypass blockcmd, orbit and all in-game punishments by relogging

Including cage.

**Comment by JeromSar** [comment]
@marcocorriero Which doesn't make it a bug. This is an exploit, and a fix for that exploit is an enhancement.
